src/main/docs/guide/appendix/architecture/compilerArch.adoc

```diff
@@ -1,7 +1,8 @@
 The Micronaut Compiler is a set of extensions to existing language compilers:
 
-* Java - the link:{jdkapi}/java.compiler/javax/annotation/processing/package-summary.html[Java Annotation Processing (APT) API] is used for Java & Kotlin code (a future version of Micronaut will support KSP for Kotlin)
-* Groovy - Groovy https://docs.groovy-lang.org/latest/html/api/org/codehaus/groovy/transform/ASTTransformation.html[AST Transformations] are used participate in the compilation of Groovy code.
+* Java - the link:{jdkapi}/java.compiler/javax/annotation/processing/package-summary.html[Java Annotation Processing (APT) API] is used for Java  code.
+* Kotlin - the https://kotlinlang.org/docs/ksp-overview.html[KSP (Kotlin Symbol Processing) API] is used for Kotlin code & https://kotlinlang.org/docs/ksp-overview.html[KAPT (Kotlin Annotation Processing Tool)] is used to support Java annotation processors in Kotlin.
```

Comment:
You are linking twice to the same link https://kotlinlang.org/docs/ksp-overview.html

Reply:
resolved via https://github.com/micronaut-projects/micronaut-core/pull/11235/commits/4ebbc7e2996c8ee31cc44c502d5ca828651e338c